src/cljs/nr/gamelobby.cljs

```diff
@@ -352,7 +352,11 @@
         [:p
          [:input.game-title {:on-change #(swap! s assoc :password (.. % -target -value))
                              :type "password"
-                             :value (:password @s) :placeholder "Password" :maxLength "30"}]]
+                             :value (:password @s) :placeholder "Password" :maxLength "30"
+                             :on-key-press (fn [e]
+                                            (when (= 13 (.charCode e))
+                                              ((join prompt))))}]]
```

Comment:
```suggestion
                                              (join prompt)))}]]
```
My apologies, I was unclear in my last suggestions.

Reply:
No, I should have guessed that.  Thank you.